docs: Updated getting started page with initial network/firewall config (#5454)
eclipse/kura: docs/getting-started/install-kura.md

### Initial network configuration

During the installation of Eclipse Kura with network management support, the initial network configuration will be generated dynamically. The existing wired and wireless network interface names are detected and sorted in ascending lexicographic order at the installation.

> If only one ethernet interface is detected (i.e. eth0), it will be configured as follows:

Reviewer: Is this an "i.e." or an "e.g"?

> The abbreviation “i.e.” stands for id est, which is Latin for “that is.” The abbreviation “e.g.” stands for the Latin phrase exempli gratia, meaning “for example.”

### Initial firewall configuration

> As done for the initial network configuration, the initial firewall setup is adjusted based on the network interface detected on the system. In the case of multiple ethernet and wireless interfaces, the configuration will be as shown in the screenshot below.

Reviewer: I would rephrase as follows

```suggestion
Similarly to the initial network configuration, the initial firewall setup is adapted based on the network interface detected on the system. In case of multiple ethernet and wireless interfaces, the configuration will be as shown in the screenshot below.
```
